### DISCUSSION TITLE
Is there a way to check if an asset is written correctly from the command line?

### DISCUSSION THREAD

**Discussion by imporoutco586** [dagster-io/dagster#23407] [Q&A] [answered]
Hi,

I noticed that after dagster instance is started, the webserver ui checks the assets in the repository and gives error feedback. Is there any way to check the assets before dagster instance is started?
​
Is there a way to check the newly written asset from the command line to see if it is correct? The asset check in the documentation seems to be written specifically for certain assets, and I want the default generic error feedback, just like the ui feedback.
​
Thank you for any answers.

**Comment by garethbrickman** [comment, marked as answer]
We don't yet have a CLI command that can check/validate definitions without loading the other Dagster services/UI. Would you mind please creating a[ feature request issue](https://github.com/dagster-io/dagster/issues/new?assignees=&labels=type%3A+feature-request&projects=&template=request_a_feature.yml) so we can track user interest for it?

**Reply by imporoutco586** [reply]
Thanks! I created one now https://github.com/dagster-io/dagster/issues/23498#issue-2454756316.

**Reply by garethbrickman** [reply]
This is implemented in: https://github.com/dagster-io/dagster/issues/2838
